runtime/doc/autocmd.txt

```diff
@@ -862,6 +862,27 @@ RecordingLeave			When a macro stops recording.
 				Sets these |v:event| keys:
 				    regcontents
 				    regname
+							*SafeState*
+SafeState			When nothing is pending, going to wait for the
```

Comment:
Could be useful for GUIs to know when `nvim_input()` was processed. Except it doesn't help if user input triggers operator-pending. Sigh.